actions/plugins/backend/action.yml

```diff
@@ -1,9 +1,18 @@
 name: Plugins - Backend - Test and build
 description: Tests, lints and builds the backend.
 
+inputs:
+  github-token:
+    description: GitHub token for downloading dependencies from private repos, if necessary
+    required: true
+
 runs:
   using: composite
   steps:
+    - name: Config git to use GitHub token
+      run: git config --global url."https://oauth2:${{ inputs.github-token }}@github.com/".insteadOf "https://github.com/"
+      shell: bash
```

Comment:
Do we also need to update `GOPRIVATE`, or is this enough? I remember reading something about it in the Slack thread :thinking: